test/test_html_reporter.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+from result.html_reporter import HTMLReporter
+
+
+class TestHTMLReporter:
+    test_junit_report_path = '/test/path/to/junit/report.xml'
```

Comment:
Yes, it's true that the fixture doesn't makes sense here since we only have one function and everything can be inside the only test case that we have so far. I did this because we plan to add other functions there for other reporting purposes. So, if this is not a blocker, I would prefer to keep it like this.